quota/etcd/quotaapi/quota_server_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quotaapi
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/google/trillian/quota/etcd/quotapb"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestServer_Unimplemented(t *testing.T) {
```

Comment:
Not really: as I implement the methods I'll naturally move things to their own tests, and this will be phased out. I'm mostly trying to keep the PRs in a manageable size by pushing a "server skeleton" first.